src/events/nats.js

```diff
@@ -55,6 +56,9 @@ const getPublishParams = (subject, msg, opt_reply, opt_callback, jsonConnectProp
         opt_callback_internal = opt_reply;
         opt_reply_internal = undefined;
     }
+    if (jsonConnectProperty && (process.env.EPSAGON_PROPAGATE_NATS_ID || '').toUpperCase() === 'TRUE') {
```

Comment:
I think we would like to use `TRUE` in default. Please check with @ranrib / @henprtz

Reply:
I talked about it with ran, He told me to use default `FALSE`